src/applications/personalization/profile360/selectors.js

```diff
@@ -1,9 +1,6 @@
 import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
 import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
 
-export const profileShowDirectDeposit = state =>
```

Comment:
removing this selector since it was only used in the ProfileView and will no longer be needed at all.